lib/client.js

```diff
@@ -7,7 +7,9 @@ import NativePromise from 'native-promise-only'
 const Promise = window.Promise || NativePromise
 const PROMISE_TIMEOUT = 10000
 // 10 seconds, see ZD#4058685
```

Comment:
// 5 mins? 🤦‍♂️

Reply:
This refers to the line above and I've not changed that value. The next line sets the longer timeout actually used in the `setTimeout` function, with an appropriate comment underneath it

Reply:
@Mehul-Porwal can I resolve this?